lib/stencil-start.js

```diff
@@ -236,6 +269,23 @@ class StencilStart {
         );
     }
 
+    async checkLangFiles(langsPath, defaultShopperLanguage) {
+        const filesPaths = await this._fsUtils.recursiveReadDir(langsPath);
+        const isDefaultLanguagePresent = filesPaths.some((file) =>
+            file.includes(defaultShopperLanguage),
+        );
+
+        if (!isDefaultLanguagePresent) {
+            this._logger.log(
+                `${
+                    'Warning'.yellow
+                }: lang folder is missing default shopper language: ${defaultShopperLanguage}`,
```

Comment:
🍹 I would maybe reword this to be something like "missing language file for default shopper language: ${defaultShopperLanguage}". I think that reads more naturally to me.